packages/cfd/src/Components/cfd-mt5-demo-account-display.tsx

```diff
@@ -0,0 +1,199 @@
+    openAccountNeededModal: (target: string, target_label: string, target_dmt5_label: string) => void;
+    standpoint: TStandPoint;
+    is_loading: boolean;
+    is_logged_in: boolean;
+    isSyntheticCardVisible: (account_category: string) => boolean;
+    isFinancialCardVisible: () => boolean;
+    onSelectAccount: (objCFDAccount: TObjectCFDAccount) => void;
+    openAccountTransfer: (
+        data: DetailsOfEachMT5Loginid | TTradingPlatformAccounts,
+        meta: TOpenAccountTransferMeta
+    ) => void;
+    platform: TCFDPlatform;
+    current_list: Record<string, DetailsOfEachMT5Loginid>;
+    openPasswordManager: (login?: string, title?: string, group?: string, type?: string, server?: string) => void;
+    residence: string;
+    landing_companies?: LandingCompany;
+    toggleMT5TradeModal: () => void;
+};
+
+const CFDMT5DemoAccountDisplay = ({
+    is_eu,
+    is_eu_country,
+    has_maltainvest_account,
+    has_cfd_account_error,
+    openAccountNeededModal,
+    standpoint,
+    is_loading,
+    is_logged_in,
+    isSyntheticCardVisible,
+    isFinancialCardVisible,
+    onSelectAccount,
+    openAccountTransfer,
+    platform,
+    current_list,
+    openPasswordManager,
+    residence,
+    toggleMT5TradeModal,
+}: TCFDDemoAccountDisplayProps) => {
+    const is_eu_user = (is_logged_in && is_eu) || (!is_logged_in && is_eu_country);
+
+    const openCFDAccount = () => {
+        if (is_eu && !has_maltainvest_account && standpoint.iom) {
+            openAccountNeededModal('maltainvest', localize('Deriv Multipliers'), localize('demo CFDs'));
+        } else {
+            onSelectAccount({
+                category: 'demo',
+                type: 'financial',
+                platform,
+            });
+        }
+    };
+
+    const financial_accounts_data = () => {
+        const acc = Object.keys(current_list).some(key => key.startsWith(`${platform}.demo.financial`))
+            ? Object.keys(current_list)
+                  .filter(key => key.startsWith(`${platform}.demo.financial`))
+                  .reduce((_acc, cur) => {
+                      _acc.push(current_list[cur]);
+                      return _acc;
+                  }, [] as DetailsOfEachMT5Loginid[])
+            : undefined;
+        return acc;
+    };
+
+    const financial_specs = React.useMemo(() => {
+        const should_show_eu = (is_logged_in && is_eu) || (!is_logged_in && is_eu_country);
+        if (residence === 'au') {
+            return specifications[platform as keyof TSpecifications].au_real_financial_specs;
+        }
+        if (should_show_eu) {
+            return specifications[platform as keyof TSpecifications].eu_real_financial_specs;
+        }
+        return specifications[platform as keyof TSpecifications].real_financial_specs;
+    }, [is_logged_in, is_eu, is_eu_country, residence, platform]);
+
+    return is_loading ? (
+        <div className='cfd-demo-accounts-display'>
+            <Loading />
+        </div>
```

Comment:
```suggestion
        <div className='cfd-demo-accounts-display'>
            {is_loading ? 
            <Loading /> : ..... }
        </div>
```
Since the container div is actually the same for both cases, you can actually nest the condition within it instead of outside and repeating the div.

Reply:
Sorted.